cts/cts-fencing.in

```diff
@@ -168,6 +169,24 @@ def killall(process):
     test.wait()
 
 
+def is_any_proc_running(process_names=[]):
```

Comment:
Currently not used directly. Planning to use the same scheme in `cts-exec`, where we do check for multiple processes.

It seems like pulling a lot stuff into a shared cts lib would be nice. In fact I think we've discussed that as a future low-priority goal.